umm-spec-lib/src/cmr/umm_spec/test/expected_conversion.clj

```diff
@@ -445,7 +445,10 @@
 (defmethod umm->expected-convert :iso-smap
   [umm-coll _]
   (let [original-brs (get-in umm-coll conversion-util/bounding-rectangles-path)
+        original-platforms (:Platforms umm-coll)
```

Comment:
Ok thanks, weird we do the 19115 convert, but that is for another day another ticket.